gulpfile.js

```diff
@@ -61,14 +61,6 @@ var TEST_SETS = [
 // Ignore ts-compile errors while watching (but not in normal builds).
 var watching = false;
 
-var TS_SETTINGS = {
-  sortOutput: true,
-  declarationFiles: true,
```

Comment:
ASFAIK we don't save the .d.ts files so we don't need declarationFiles to be generated. 
The others are not actual compiler options.
sortFiles is used to preserve source-file maps across browserfy but it seems to works without this flag.
I am not sure why we had noExternalResolve becuase it also all seems to work without it. Do you know why we have them?